array-manipulation-1/src/tail.js

```diff
@@ -0,0 +1,17 @@
+/* exported tail */
+
+// create storage for individual items
+// create storage for new list of items
+// take out items from list one by one except for the first items
+// put extracted items back into list
+// give back new list
+
+function tail(array) {
+  var item = '';
+  var newArray = [];
+  for (var i = 0; i < array.length - 1; i++) {
+    item = array[i + 1];
```

Comment:
If you just start the loop at `1`, you can just use the value of `i`, not `i + 1` 🤓